database/things/koka.pldb

```diff
@@ -6,7 +6,7 @@ description A strongly typed functional-style language with effect types and han
 website https://www.microsoft.com/en-us/research/project/koka/?from=http%3A%2F%2Fresearch.microsoft.com%2Fen-us%2Fprojects%2Fkoka
 webRepl https://rise4fun.com/koka/
 reference https://koka-lang.github.io/koka/doc/book.html
-originCommunity Microsoft
+originCommunity Microsoft Research
 
```

Comment:
country United States
Daan Leijen is based in Redmond, Washington
https://www.microsoft.com/en-us/research/people/daan/